gui/city_info.h

```diff
@@ -77,6 +84,15 @@ class city_info_t : public gui_frame_t, private action_listener_t
 
 	void update_data();
 
+	// update pointer to this->city
+	void update(stadt_t*& c) { c = city; };
+
+	// get city pointer
+	stadt_t* get_city() { return city; };
+	
+	// is highlighted?
+	bool get_highlight() { return highlighted; };
```

Comment:
Can't this be like this?
```suggestion
	bool is_highlighted() { return highlight.pressed; };
```